src/components/ReportActionItem/MoneyReportView.js

```diff
@@ -26,17 +28,24 @@ const propTypes = {
 };
 
 function MoneyReportView(props) {
+    const {isSmallScreenWidth} = useWindowDimensions();
     const formattedAmount = CurrencyUtils.convertToDisplayString(ReportUtils.getMoneyRequestTotal(props.report), props.report.currency);
     const isSettled = ReportUtils.isSettled(props.report.reportID);
     const {translate} = useLocalize();
 
+    // If window width is greater than the max background width, repeat the background image
+    const maxBackgroundWidth = variables.sideBarWidth + CONST.EMPTY_STATE_BACKGROUND.ASPECT_RATIO * CONST.EMPTY_STATE_BACKGROUND.WIDE_SCREEN.IMAGE_HEIGHT;
+    const platform = getPlatform();
+    const isWebPlatform = [CONST.PLATFORM.WEB, CONST.PLATFORM.DESKTOP].includes(platform);
```

Comment:
@aswin-s Why do we need platforms check? Regardless of platform the solution is specific to image size and screen size, right?

Reply:
@abdulrahuman5196  We are using Image component from 'react-native-fast-image' in these screens. FastImage doesn't support 'repeat' resize mode. So I just limited it to Web based platforms. The other option is to use Image component from 'react-native'. If you think that's ok I'll make the switch and remove the platform checks.

Reply:
Oh I just checked again. Fast image is used only on 'MoneyRequestView' page. I'll switch out the component in that screen and remove the check all together.